cmd/flags_firewall.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package cmd
+
+import (
+	"github.com/mysteriumnetwork/node/core/node"
+	"github.com/urfave/cli"
+)
+
+var (
+	enableKillSwitch = cli.BoolFlag{
+		Name:  "firewall.killSwitch",
+		Usage: "Enable consumer outgoing non tunneled traffic blocking during connections",
```

Comment:
Do we really need as a flag here? IMHO it is a sensible out of the box behaviour. You launch VPN - everything goes through it. This flag would only puzzle our users.

Reply:
Its still experimental. I wrote args for you and @Waldz, actually I am ok to remove it ant leave Noop for other OSes.

Reply:
Lets remove experimental flag

Reply:
Removed